packages/components/inputs/multiline-text-input/src/multiline-text-input.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { type TMultilineTextInputProps } from './multiline-text-input';
+
+const getMultilineTextInputActionIconStyles = (
+  props: TMultilineTextInputProps
+) => css`
+  position: absolute;
+  right: ${designTokens.spacing30};
+  top: ${props.isCondensed ? '7px' : designTokens.spacing20};
```

Comment:
If you want to vertically center the icon within the boundaries of the first row, the following works as well and without a magic number:

```suggestion
top: 0;
  height: ${props.isCondensed
    ? `${designTokens.heightForInputAsSmall}`
    : `${designTokens.heightForInput}`};
  padding: 0;
  display: flex;
  justify-content: center;
  align-items: center;
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2845/commits/022f3b1b0ff8a0368ef2cad6b3cd9f2867eb1491
Thank you